packages/driver/src/cy/multi-domain/util.ts

```diff
@@ -17,3 +18,30 @@ export const correctStackForCrossDomainError = (serializedError: any, userInvoca
 
   return reifiedError
 }
+
+export const omitUnserializableEnvAndConfigValues = (value) => {
+  const { isDom, isDocument, isWindow, isJquery } = $dom
+
+  // There are probably some things we aren't catching here.
+  // We probably want to bubble this up to the user over a generic error to get a better idea of what couldn't be serialized
+  if (isDom(value)
+  || isDocument(value)
```

Comment:
I was hoping to do something similar to the [subject](https://github.com/cypress-io/cypress/pull/19936/files#diff-a4bcf3a0aee37eb48297716f3b3fc07df008c190d68af82e29eda0e65ccf9c1aR5) work, but we can't really leverage a proxy here since config creates getters and setters under the hood and would ultimately throw an error anytime something is not serializable. Is there anything we are missing here that we know about?